dCommon/Game.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <iosfwd>
```

Comment:
Why iosfwd for a new std::string value? I can't see how that'd forward declare string

Reply:
I confused std::string with std::stringstream declaration